### DISCUSSION TITLE
Emojis render much larger than surrounding text

### DISCUSSION THREAD

**Discussion by myang7** [dunst-project/dunst#1248] [Q&A]
### Issue description
When a notification from dunst contains an emoji, the emoji is multiple times larger than the surrounding text
![2023-06-08-133821_353x181_scrot](https://github.com/dunst-project/dunst/assets/65783258/e7b3d0e4-d15a-4c2a-bb24-088edef1375b)

### Installation info

- Version: `1.9.2`
- Install type: `package`
- Window manager / Desktop environment: `dwm`
- Distro: `Gentoo`

<details>
<summary>Minimal dunstrc</summary>

[global]
font = Hack Nerd Font
</details>

**Comment by gnusenpai** [comment]
Also on Gentoo; I discovered that enabling `10-scale-bitmap-fonts.conf` using eselect fontconfig fixed the issue for me.
